Etabs_Adapter/CRUD/Delete/_Delete.cs

```diff
@@ -16,7 +17,7 @@ public partial class ETABS2016Adapter : BHoMAdapter
         /**** Public Methods                            ****/
         /***************************************************/
 
-        protected override int Delete(Type type, IEnumerable<object> ids)
+        protected override int IDelete(Type type, IEnumerable<object> ids, ActionConfig actionConfig = null)
```

Comment:
Keeping the file, as this will need to get implemented sooner or later. Have changed it to call the base method instead though to make use of the improved error messaging.